docker/server/entrypoint.sh

```diff
@@ -61,9 +64,31 @@ do
     fi
 done
 
+# if clickhouse user is defined - create it (user "default" already exists out of box)
+if [ -n "$CLICKHOUSE_USER" ] && [ "$CLICKHOUSE_USER" != "default" ] || [ -n "$CLICKHOUSE_PASSWORD" ]; then
+    echo "Create user '$CLICKHOUSE_USER' instead default"
+    cat <<EOT >> /etc/clickhouse-server/users.d/default-user.xml
```

Comment:
That is entrypoint that code will be executed also on containter restart (adding more lines to default-user.xml every time)

try `docker restart container_name`

Reply:
You should wither adjust that conditionally, or just overwrtewhole file each time ( `>` instead of `>>`)

Reply:
Oh, my bad again, fixed